code/clue.js

```diff
@@ -1,59 +1,237 @@
-// STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
+const Marcel = {
+  firstName: 'Marcel',
+  lastName: 'the Monkey',
+  description: 'He was born to steal',
+  age: 3,
+  image: 'assets/marcel.png',
+  occupation: 'Monkey',
+  favouriteFood: 'Banana'
+};
 
-const mrGreen = {
-  firstName: 'Jacob',
+const Ross = {
+  firstName: 'Ross',
+  lastName: 'Geller',
+  description: 'A palaeontologist working at the Museum of Natural History and in love with Rachel.',
+  age: 27,
+  image: 'assets/ross.png',
+  occupation: 'Palaeontologist',
+  favouriteFood: 'Muffin'
+};
+
+const Chandler = {
+  firstName: 'Chandler',
+  lastName: 'Muriel Bing',
+  description: 'Making jokes whe he feels himself uncomfortable and loves Monica.',
```

Comment:
Spoiler alert!

Reply:
If you have ever seen just one minute of friends it soo obvious that they are in love, even the scene is not about Monica and Chandler:D